static/templates/index.html

```diff
@@ -40,10 +40,21 @@
 		</p>
 	</div>
 	{{ if gt .StargazersCount 0 }}
+	<div>
+		<div class="button-group">
+            <button data-variant="adoptive" class="active">Adoptive</button>
+            <button data-variant="light">Light</button>
+            <button data-variant="dark">Dark</button>
+            <button data-variant="custom">Custom</button>
+		</div>
+
+	</div>
 	<div class="chart">
 		<p>
-			<img src="/{{ .FullName }}.svg"
-				alt="Please try again in a few minutes. This might not work for very famous repository.">
+			<img src="/{{ .FullName }}.svg?variant=adoptive"
```

Comment:
```suggestion
			<img src="/{{ .FullName }}.svg?variant=adaptive"
```

Reply:
Sorry, I have fixed these mistakes in the next commits. My pr still in progress, I needed a little more time to finish.